Tests/RegexBuilderTests/AlgorithmsTests.swift

```diff
@@ -38,13 +40,13 @@ class RegexConsumerTests: XCTestCase {
       int,
       input: "foo 160 bar 99 baz",
       result: "foo 240 bar 143 baz",
-      { match in String(match.result.1, radix: 8) })
+      { match in String(match.output.1, radix: 8) })
```

Comment:
Just to make sure, couldn't this be `match.1`?, etc? No need to synthesize the entire output.

Reply:
Yeah you're right that this can just be `match.1`. I was just doing a blind replace.